src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/EnergyConnector.java

```diff
@@ -28,10 +28,18 @@
  *
  */
 public class EnergyConnector extends SimpleSlimefunItem<BlockUseHandler> implements EnergyNetComponent {
-
+    private final int RANGE;
     @ParametersAreNonnullByDefault
     public EnergyConnector(ItemGroup itemGroup, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe, ItemStack recipeOutput) {
         super(itemGroup, item, recipeType, recipe, recipeOutput);
+
+        this.RANGE = 6;
```

Comment:
just call the other constructor
```suggestion
        this(itemGroup, item, recipeType, recipe, recipeOutput, 6);
```